Source/Core/WebService.swift

```diff
@@ -13,28 +13,68 @@ import Foundation
  and processing the resulting `NSURLResponse` object.
 */
 @objc public final class WebService: NSObject {
-    /// Base URL of the web service.
-    public let baseURLString: String
-    
+    /**
+        Base URL of the web service.
+        If the base URL is nil, the path is interpreted as an absolute URL.
+     */
+    private (set) public var baseURL: URL? = nil
```

Comment:
It is the same as the existing use case of setting the `baseURLString` to "".  e.g using relative paths that are full URL strings.